packages/core/src/domain/configuration/transportConfiguration.ts

```diff
@@ -26,13 +26,13 @@ export function computeTransportConfiguration(initConfiguration: InitConfigurati
   const tags = buildTags(initConfiguration)
 
   const endpointBuilders = computeEndpointBuilders(initConfiguration, tags)
-  const intakeUrlPrefixes = computeIntakeUrlPrefixes(endpointBuilders, site)
```

Comment:
**💬 suggestion:** ‏Looks like URL prefixes are not used anymore, could you remove them?